cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -109,10 +117,16 @@
             padding-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
         }
 
-        .m-global-eyebrow_languages {
-            padding-left: 37px; //22px flag + 15px spacing
+        .m-global-eyebrow_right {
             padding-top: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
             border-top: 1px solid @gray-40;
+            text-align: left;
+        }
+
+        .m-global-eyebrow_phone,
+        .m-global-eyebrow_languages {
+          display: block;
+          padding-left: 37px; //22px flag + 15px spacing
```

Comment:
Both the `_phone` and `_languages` have left padding that needs to be overwritten. Including it here handles that and adds the spacing.